canonical/partners.ubuntu.com
# Added github actions for pull requests

## Done

- Added github actions for partners.u.c

## QA

1. On this PR, confirm that all checks are running and have passed. 
- run-image
- run-dotrun
- lint-python
- validate-deploy
- test-python
- check-inclusive-naming 

## Issue / Card

Fixes #281
